### ISSUE TITLE
jumping tabs left and right

### ISSUE THREAD

**Issue by NickIgnat** [crittermike/shortkeys#512]
is there functionality to customise jumping left and right in opened tabs? The default is command+option+(left/right)

**Comment by Aetherinox** [comment]
You guys ever find a better solution?

I can get it to work for about 90% of tabs. But it will get stuck if you haven't reloaded that page, or on certain websites like Mozilla.

**Comment by hababr** [comment]
@Aetherinox Try to setup shortcuts for the extension in the browser's settings. This way they can be triggered from any page.
![shkeys](https://github.com/user-attachments/assets/9803b27d-fd6a-4ec0-85dc-ec343a34d5c0)

**Comment by Aetherinox** [comment]
Is that the same place I was at?

At present I configured the plugin itself in Firefox:

![X7U6hLZN4Q](https://github.com/user-attachments/assets/df105bee-9d6f-4ed5-acd9-4460c8ef6c65)

**Comment by hababr** [comment]
>This will work good. Only downside is that it won't let me use just the arrow keys left / right. It wants CTRL or ALT.

Yes, this is the settings I meant. I use Alt+1 and Alt+2, that way I can press it with one hand and it is similar to Alt+Tab.

**Comment by Aetherinox** [comment]
Yeah, both could be useful. Sometimes I'm working and I use my right hand to just hit the arrow keys.
I've also got it set up to use my mouse and I just tap the button.

I forgot I had this installed. Today I was in Jellyfin watching a movie, and I wanted to skip ahead because I re-loaded the page by accident. I still had the old arrow key only bind set up, so I pressed the right arrow, and Firefox went "Weeeeeeeeeeee" and flashed me through about 8 tabs in a second or two.

Rather interesting.
